README.md

```diff
@@ -37,6 +37,35 @@ doctrine:
                     engine: MyISAM
 ```
 
+## MySQL Strict Mode
+
+It is recommended to run MySQL in "strict mode" to prevent corrupt or truncated
+data and to guarantee data integrity. As of Contao 4.12, the install tool shows
+a warning if the database server is not running in strict mode. To enable it,
+add the following to your `my.cnf` file or make sure that the setting is adjusted
+accordingly:
+
+```
+[mysqld]
+…
+sql_mode="STRICT_TRANS_TABLES"
```

Comment:
```suggestion
sql_mode="TRADITIONAL"
```

We should probably suggest the `TRADITIONAL` mode here too as we do in the connection settings.

Reply:
> We should probably suggest the `TRADITIONAL` mode here too as we do in the connection settings.

But the hint in the install tool says "Bitte aktivieren Sie den **Strict-Mode** entweder in Ihrer `my.cnf`-Datei …", so I thought it would be recommended to configure the operation mode to the strict one in the first place. Of course, this assumes that you actually have access to the `my.cnf` or `my.ini` MySQL configuration file.

Reply:
`TRADITIONAL` already includes the following:

```
STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
```

`TRADITIONAL` is the default and recommended setting.

Reply:
Thank you, then of course it makes sense. 👍🏻